vars/RedHat_6.yml

```diff
@@ -1,4 +1,4 @@
-timesync_ntp_provider_os_default: "ntp"
+timesync_ntp_provider_os_default: "chrony"
```

Comment:
In RHEL 6 the default was ntpd (in the sense that it was installed by default, chrony was not, although it was available).

Reply:
Yup, meant to revert this. We just want to make chrony the default for our environment.